oxen/pyproject.toml

```diff
@@ -20,6 +20,8 @@ dependencies = [
     "requests",
     "toml",
     "tqdm",
+    "torch",
+    "tensorflow",
```

Comment:
I believe this will make package managers try to install torch and tensorflow as dependencies for oxen. Is that what we want?

Reply:
I was debating, because they are used in our data loaders (that are undocumented) but are a heavy install. Maybe we do just keep them in the dev dependencies for now...